### PR TITLE
reorganize migration section

### PR DESCRIPTION
**Pull request by emuus**
-remove ref to whitebox and black box
-focus on new terms : lift & shift and re-platform
-merge duplicate info
-streamline info (removing info that is too detailed)

### CODE REVIEW

File: end-user/en/source/pages/migration/overview.rst [hunk 1]
```diff
@@ -7,22 +7,23 @@ Migrating Live Workloads
 
 UForge AppCenter offers the capability to migrate a live system using two different methods:
 
-* using ``Re-platform``. In this case there are three steps to migration: scan, generate and publish. Each step is done seperately by the user. When using ``Re-platform`` to migrate, you can modify the appliance prior to the final migration.
-* using ``Lift & Shift``. This is an automated process.
+* using ``Lift & Shift``. This is an automated process which covers three steps: scan, generate and publish.
+* using ``Re-platform``. In this case there are three or four steps executed seperately by the user: scan, generate and publish. When using ``Re-platform`` to migrate, you can import the appliance and modify it prior to the final migration.
```
Comment (jgweir) [inline end-user/en/source/pages/migration/overview.rst]: You mention 4 steps, but there are only three with an extra sentence describing the fourth.  Should we not use: scan, update (or edit), generate and publish.

File: end-user/en/source/pages/migration/overview.rst [hunk 2]
```diff
@@ -7,22 +7,23 @@ Migrating Live Workloads
...
 
 In both cases, UForge will "deep scan" a live system and report back the meta-data of every file and package that makes up the running workload.
 
+The goal migration (either using ``Re-platform`` or ``Lift & Shift``) is to reproduce a near identical copy of the currently running workload.  However, there will always be small differences between the two workloads after migration is complete, notably some services are disabled or enabled depending on the target machine image being created (refer to :ref:`service-state`). For more information on the differences after migration, refer to :ref:`source-target-diffs`.
```
Comment (jgweir) [inline end-user/en/source/pages/migration/overview.rst]: Actually for replatform, the goal is to actively make changes.  I would say the goal for Lift and Shift is to create a near identical copy.  For replatform the goal is to allow the user to make the necessary changes (potentially add or remove software, upgrade, re-configure etc)

File: end-user/en/source/pages/migration/migration-prereq.rst [hunk 3]
```diff
@@ -5,12 +5,35 @@
 Migration Pre-requisites
 ------------------------
 
+You must have root access on the target system in order to complete a migration, as you will need to copy and run a binary file on the target system.
```
Comment (jgweir) [inline end-user/en/source/pages/migration/migration-prereq.rst]: There are a lot of warnings and notes (that have been added through time) should we not group these together or at least have all the notes together).  Just a thought

File: end-user/en/source/pages/migration/process-overview.rst [hunk 4]
```diff
@@ -5,17 +5,52 @@
...
    2. The scan report and overlay is sent to UForge AppCenter.
-   3. A machine image is generated from the scan (black box migration).  With ``Re-Platform`` you can also import the scan prior to machine image generation to change the content (white box migration).
+   3. A machine image is generated from the scan.  With ``Re-Platform`` you can import the scan prior to machine image generation to change the content.
    4. The machine image is uploaded and registered to the target environment.
    5. The registered machine image (also known as a template in cloud terminology) can be used to provision one or more instances.  These instances have near identical content to the original live system.
 
    .. image:: /images/migration-process.jpg
+
+
+For both migration using ``Lift & Shift`` and ``Re-platform``, UForge will scan the system you want to migrate and detect the following information:
```
Comment (jgweir) [inline end-user/en/source/pages/migration/process-overview.rst]: After the 5 steps, I would explicitly mention that "Lift and Shift" automatically executes steps 1, 2, 3 and 4.  For "replatform" each of these steps are done manually by the user allowing the person to make the necessary changes after step 2.

Comment (emuus) [reply end-user/en/source/pages/migration/process-overview.rst]: done

File: end-user/en/source/pages/migration/overview.rst (hunk 2 above)
Comment (emuus) [reply end-user/en/source/pages/migration/overview.rst]: good point

File: end-user/en/source/pages/migration/overview.rst (hunk 1 above)
Comment (emuus) [reply end-user/en/source/pages/migration/overview.rst]: good point

File: end-user/en/source/pages/migration/migration-prereq.rst (hunk 3 above)
Comment (emuus) [reply end-user/en/source/pages/migration/migration-prereq.rst]: reorg the notes and warnings into sub-sections to help readability

**Pull request merged**